CONTRIBUTING.md

```diff
@@ -48,6 +49,12 @@ The Redwood Framework lives in the monorepo `redwoodjs/redwood` (which is where
 
 Throughout this document, we'll assume your local copy of the Redwood Framework is in a directory called `redwood` and your Redwood App is in a directory called `redwood-app`.
 
+### Browser-based Setup
+
+Use the following button to start a development environment in the cloud and access it through your browser.
```

Comment:
```suggestion
Use the following button to start a development environment in the cloud and access it through your browser.

This will generate a test blog project, and link it with the framework code in `main` - and gives you an easy playground to try out your fixes and contributions.

> Note: if you make changes to the framework, you will need to run `yarn rwfw project:sync` in the terminal, so that your changes are watched and reflected in the test project
```